analyses/fusion_filtering/04-project-specific-filtering.Rmd

```diff
@@ -88,8 +96,13 @@ fusion_calls<-QCGeneFiltered_filtFusion %>% mutate(FusionName=rm_between(.data$F
 group<-params$group
 
 # get histology file
+if ( params$base_run ==0 ){
```

Comment:
Looks like the only difference here is whether `params$base_histology` or `params$histology` is used, so I would reduce your if statement to specify a `histology_file` variable and then supply that to that code chunk rather than repeating the code chunk twice.